components/Card.js

```diff
@@ -22,7 +22,7 @@ export default function Card({ image, title, year }) {
     <>
       <CardImage src={image} alt={title} width={180} height={250} />
```

Comment:
This is not directly related to the user story, but I just realized that you add other width and height here than in the styled CardImage component. Is that intended?